app_widgetbook/lib/main.dart

```diff
@@ -343,6 +346,35 @@ class WidgetbookApp extends StatelessWidget {
                     ),
                   ],
                 ),
+                WidgetbookComponent(
+                  name: 'AppIcon',
+                  useCases: List.generate(
+                    AppIcons.iconPaths.length,
+                    (index) => WidgetbookUseCase(
+                      name: AppIcons.iconPaths[index]
+                          .replaceAll('assets/images/icons/', '')
+                          .replaceAll('.png', ''),
```

Comment:
This looks pretty funny to obtain a name 😀 Maybe a `RegEx` is more appropriate? 

But in the end, I think such an implementation is not possible with the current implementation of `widgetbook_generator`. Therefore, you can leave it as it is right now.

Reply:
Did a minor change on this 06be5d9929a1260e5ab872b1a3bf63a8c0f9a37c.